code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -4,28 +4,28 @@
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "bandaid"
 	item_state = "bandaid"
-	possible_transfer_amounts = list()
 	volume = 40
 	apply_type = PATCH
 	apply_method = "apply"
 	self_delay = 30		// three seconds
 	dissolvable = FALSE
 
 /obj/item/reagent_containers/pill/patch/attack(mob/living/L, mob/user)
-	if(ishuman(L))
-		var/obj/item/bodypart/affecting = L.get_bodypart(check_zone(user.zone_selected))
-		if(!affecting)
-			balloon_alert(user, "The limb is missing.")
-			return
-		if(!IS_ORGANIC_LIMB(affecting))
-			balloon_alert(user, "[src] doesn't work on robotic limbs.")
-			return
-	..()
+	if(!ishuman(L))
+		return ..()
+	var/obj/item/bodypart/affecting = L.get_bodypart(check_zone(user.zone_selected))
+	if(!affecting)
+		balloon_alert(user, "The limb is missing.")
+		return
+	if(!IS_ORGANIC_LIMB(affecting))
+		balloon_alert(user, "[src] doesn't work on robotic limbs.")
+		return
+	return ..(L, user, affecting)
```

Comment:
why is the super call implementing the affecting argument but this proc only has the two?